src/ztex/fpga-sha256crypt/README.md

```diff
@@ -0,0 +1,143 @@
+## Overview
+
+- sha256crypt for ZTEX 1.15y board allows candidate passwords up to
+32 bytes, equipped with on-board mask generator and comparator.
+The board computes 600 keys in parallel.
+
+
+## Computing units
+
+- sha256crypt invokes SHA256 in different ways, sometimes that look
+very complex. To accomplish the task, a CPU-based computing unit
+was implemented.
+- Each unit consists of 3 cores, CPU, memory and I/O subsystem.
+The approximate schematic of a computing unit is shown at fig.1.
+
+```
+           --------------+-------------+--------------
+          /             /             /               \
+     +--------+    +--------+    +--------+            |
+     |        |    |        |    |        |            |
+     | SHA256 |    | SHA256 |    | SHA256 |            |
+     |  core  |    |  core  |    |  core  |            |
+     |   #0   |    |   #1   |    |   #2   |            |
+     |        |    |        |    |        |            |
+     +--------+    +--------+    +--------+            |
+         ^            ^            ^                   |
+          \           |           /                    |
+           +----------+-----------                     |
+           |                                           |
+ . . . . . | . . . . . . . . . . . . . . . . . . . . . |. .
+           |                                          /
+           |                                         /
+  +---------------+         -------------           /
+  |               |        /             \         /
+  | process_bytes |       /    "main"     \ <------
+  |               |<--+--|      memory     |
+  +---------------+   |   \    (16x256b)  / <------
```

Comment:
Should be 6x128B. BTW, let's use "B" for bytes if we use "b" for bits (ditto for the sha512crypt-ztex README). These documentation issues can be fixed in a separate commit and PR for both sha256crypt-ztex and sha512crypt-ztex.